python/py-scipy/Portfile

```diff
@@ -6,11 +6,11 @@ PortGroup               active_variants 1.1
 PortGroup               github 1.0
 PortGroup               compilers 1.0
 
-github.setup            scipy scipy 1.1.0 v
+github.setup            scipy scipy 1.2.0 v
 revision                1
```

Comment:
The `revision` line needs to be set to 0 when increasing the version.

Reply:
@mf2k, committed and squashed.